pkg/ctl/cmdutils/nodegroup_filter_test.go

```diff
@@ -0,0 +1,126 @@
+package cmdutils_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha4"
+
+	. "github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+)
+
+var _ = Describe("nodegroup filter", func() {
+
+	newClusterConfig := func() *api.ClusterConfig {
```

Comment:
These lambdas are duplicated, but I don't mind some amount duplication in tests, it's convenient to be able to change these in the same place.

Reply:
I'm ok with some duplication in tests, it can be better for readability. Do they need to be lambdas though? Can we move them to actual functions to make the tests a bit cleaner?

Reply:
yeah, happy to move these